fileicons/openhab-icon-theme.json

```diff
@@ -0,0 +1,150 @@
+        },
+        "_folder_html": {
+            "iconPath": "./images/Folder_16x_Html.png"
+        },
+        "_folder-open_html": {
+            "iconPath": "./images/FolderOpen_16x_Html.png"
+        },
+        "_file_items": {
+            "iconPath": "./images/File_16x_Items.png"
+        },
+        "_folder_items": {
+            "iconPath": "./images/Folder_16x_Items.png"
+        },
+        "_folder-open_items": {
+            "iconPath": "./images/FolderOpen_16x_Items.png"
+        },
+        "_file_persistence": {
+            "iconPath": "./images/File_16x_Persistence.png"
+        },
+        "_folder_persistence": {
+            "iconPath": "./images/Folder_16x_Persistence.png"
+        },
+        "_folder-open_persistence": {
+            "iconPath": "./images/FolderOpen_16x_Persistence.png"
+        },
+        "_file_rules": {
+            "iconPath": "./images/File_16x_Rules.png"
+        },
+        "_folder_rules": {
+            "iconPath": "./images/Folder_16x_Rules.png"
+        },
+        "_folder-open_rules": {
+            "iconPath": "./images/FolderOpen_16x_Rules.png"
+        },
+        "_file_scripts": {
+            "iconPath": "./images/File_16x_Scripts.png"
+        },
+        "_folder_scripts": {
+            "iconPath": "./images/Folder_16x_Scripts.png"
+        },
+        "_folder-open_scripts": {
+            "iconPath": "./images/FolderOpen_16x_Scripts.png"
+        },
+        "_file_sitemaps": {
+            "iconPath": "./images/File_16x_Sitemaps.png"
+        },
+        "_folder_sitemaps": {
+            "iconPath": "./images/Folder_16x_Sitemaps.png"
+        },
+        "_folder-open_sitemaps": {
+            "iconPath": "./images/FolderOpen_16x_Sitemaps.png"
+        },
+        "_file_sounds": {
+            "iconPath": "./images/File_16x_Sounds.png"
+        },
+        "_folder_sounds": {
+            "iconPath": "./images/Folder_16x_Sounds.png"
+        },
+        "_folder-open_sounds": {
+            "iconPath": "./images/FolderOpen_16x_Sounds.png"
+        },
+        "_file_things": {
+            "iconPath": "./images/File_16x_Things.png"
+        },
+        "_folder_things": {
+            "iconPath": "./images/Folder_16x_Things.png"
+        },
+        "_folder-open_things": {
+            "iconPath": "./images/FolderOpen_16x_Things.png"
+        },
+        "_file_transform": {
+            "iconPath": "./images/File_16x_Transform.png"
+        },
+        "_folder_transform": {
+            "iconPath": "./images/Folder_16x_Transform.png"
+        },
+        "_folder-open_transform": {
+            "iconPath": "./images/FolderOpen_16x_Transform.png"
+        }
+    },
+    "file": "_file",
+    "folder": "_folder",
+    "folderExpanded": "_folder-open",
+    "fileExtensions": {
+        "html": "_file_html",
+        "png": "_file_icons",
+        "svg": "_file_icons",
+        "items": "_file_items",
+        "sitemap": "_file_sitemaps",
```

Comment:
No, the ESH Designer does not have a services icon:
[http://docs.openhab.org/installation/images/smarthome-designer-20.png](http://docs.openhab.org/installation/images/smarthome-designer-20.png)

I can add one from the icon package, any preferences?
[http://www.famfamfam.com/lab/icons/silk/previews/index_abc.png](http://www.famfamfam.com/lab/icons/silk/previews/index_abc.png)